Troubleshooting for the missing `execute` in some function/action imports. (#236)
SAP/cloud-sdk: docs/js/features/odata/common/function-imports/entity-return-type.mdx

:::note Troubleshooting

> For some function/action-imports, the `execute()` method might be missing as intended.

Reviewer: ```suggestion
For some OData function or action imports, the `execute()` method might be missing as intended.
```

> Typically, this might happen when an entity type is shared by multiple entity sets and is used as the return type of function/action imports.

Reviewer: ```suggestion
Typically, this might happen when an entity type is shared by multiple entity sets and is used as the return type of OData function or action imports.
```

> You can then use `executeRaw` for getting the raw response and deserialize by your own.

Reviewer: ```suggestion
In such a case you can use `executeRaw` method for getting the raw response returned after invoking the OData function or action via type-safe API and deserialize it on your own.
```

Reviewer: I hope I guessed correctly that everywhere above you're talking about OData functions and actions? Also, the last one means calling our type-safe API for request building but using execute raw for a full response, right?

Author: Yes, your understanding is correct.
The "odata" are omitted as the documentation is under odata section/context, but adding them explicitly makes it less confusing.
I'll commit with the suggestions.
